v201806/adwords_user_list.go

```diff
@@ -188,7 +192,11 @@ func NewCrmBasedUserList(name, description string, membershipLifeSpan int64, opt
 }
 
 func NewMutateMembersOperand() *MutateMembersOperand {
-	return &MutateMembersOperand{DataType: "EMAIL_SHA256"}
```

Comment:
previously this was used to indicate to the API which of the identifiers we wanted to use, is this now not needed anymore?

Reply:
Correct. This API version only accepts hashed PII 🔐.